pagopa/pn-frontend
# Removed validation check for payment methods

## Short description
Removed validation check for payment methods.

## List of changes proposed in this pull request
- Step 4, Payment methods file upload are no longer required
 
## How to test
Create a new notification with at least one payment method in Step 1; fill Step 2 and 3 as usual; Step 4 leave empty. You should be able to click "create" button anyway.
